Server/Plugins/APIDump/Classes/Geometry.lua

```diff
@@ -1088,6 +1108,36 @@ aborted and Trace() returns false.</p>
 	<td>Called when the ray enters a chunk that is not loaded. This usually means that the tracing is aborted.
 	Unlike the other callbacks, this function doesn't have a return value.</td></tr>
 </table>
+</p>
+				]],
+			},
+			{
+				Header = "Deprecated Callbacks",
+				Contents = [[
+<p>The Callbacks in the Trace() function is a table that contains named functions. Cuberite will call
+individual functions from that table for the events that occur on the line - hitting a block, going out of
+valid world data etc. The following table lists all the available callbacks. If the callback function is
+not defined, Cuberite skips it. Each function can return a bool value, if it returns true, the tracing is
+aborted and Trace() returns false.<br>
+Note: The folowings are deprecated. They should be replaced with the alternatives that uses Vector3d. 
+They can only be used when the Trace() function uses double too.</p>
+<p>
+<table><tr><th>Name</th><th>Parameters</th><th>Notes</th></tr>
+<tr><td>OnNextBlock</td><td>BlockX, BlockY, BlockZ, BlockType, BlockMeta, EntryFace</td>
+	<td>Called when the ray hits a new valid block. The block type and meta is given. EntryFace is one of the
+	BLOCK_FACE_ constants indicating which "side" of the block got hit by the ray.</td></tr>
+<tr><td>OnNextBlockNoData</td><td>BlockX, BlockY, BlockZ, EntryFace</td>
+	<td>Called when the ray hits a new block, but the block is in an unloaded chunk - no valid data is
+	available. Only the coords and the entry face are given.</td></tr>
+<tr><td>OnOutOfWorld</td><td>X, Y, Z</td>
+	<td>Called when the ray goes outside of the world (Y-wise); the coords specify the exact exit point. Note
+	that for other paths than lines (considered for future implementations) the path may leave the world and
+	go back in again later, in such a case this callback is followed by OnIntoWorld() and further
+	OnNextBlock() calls.</td></tr>
+<tr><td>OnIntoWorld</td><td>X, Y, Z</td>
+	<td>Called when the ray enters the world (Y-wise); the coords specify the exact entry point.</td></tr>
+</table>
+</p>
```

Comment:
```suggestion
</table>
```

Reply:
This `</p>` still needs to be removed.

Reply:
Okay I  removed it. But id do not understand why I thought html needed a </p> for each <p>

Reply:
Yes, it does, but the APIDump plugin adds the starting `<p>` and ending `</p>` automatically. Basically it does `file:write("<p> .. AdditionalInfo[i].Contents .. </p>")`. That way the documentation doesn't need to have the tags in each member, but it can still have multiple paragraphs if it wants to.

Reply:
Okay